canonical-web-and-design/vanilla-framework
# Remove custom syntax highlighting from home page

## Done

Removes custom syntax highlighting from home page, use built-in class names instead.

Fixes canonical-web-and-design/vanilla-squad#981


## QA

- Open [demo](https://vanilla-framework-3638.demos.haus)
- Make sure quick start demo code has syntax highlighting

## Screenshots

<img width="1312" alt="Screenshot 2021-03-19 at 16 10 29" src="https://user-images.githubusercontent.com/83575/111801897-ae5dfb00-88cd-11eb-84ab-ff0f8318ac10.png">
